packages/oss-console/src/components/Literals/helpers.ts

```diff
@@ -79,6 +80,13 @@ function processBinary(binary?: Core.IBinary | null) {
     return 'invalid binary';
   }
 
+  if (tag === 'msgpack' && binary.value) {
+    return {
```

Comment:
should we still return the tag and an undefined value when `binary.value` is undefined?